requirements.txt

```diff
@@ -1,3 +1,2 @@
 Flask==0.11.1
-requests==2.11.1
-discord.py
```

Comment:
[phony.py](https://github.com/blha303/gitlab-discord-bridge/blob/master/phony.py) needs discord.py. phony.py isn't necessary, but someone asked if I could keep the bot online between messages, so I added it to the repo.